server/settings/test.py

```diff
@@ -30,7 +30,7 @@
 if not os.path.exists(STORAGE_CONTAINER):
     os.makedirs(STORAGE_CONTAINER)
 
-CACHE_TYPE = 'simple'
+CACHE_TYPE = 'null'
```

Comment:
The version test does GET /API/v3/versions/ again to see if it actually changed, but the result is cached from the first call. I'm not sure if there's another way around that